invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -43,8 +56,29 @@ export class DeleteCommunityModal extends Component {
 
   componentWillUnmount() {
     this.cancellableDelete && this.cancellableDelete.cancel();
+    this.cancellableMembersCountFetch && this.cancellableMembersCountFetch.cancel();
+    this.cancellableRecordsCountFetch && this.cancellableRecordsCountFetch.cancel();
   }
 
+  updateCommunityMetrics = async () => {
+    this.setState({ loading: true });
+    const { community } = this.props;
+    this.cancellableMembersCountFetch = withCancel(http.get(community.links.members));
+    this.cancellableRecordsCountFetch = withCancel(http.get(community.links.records));
```

Comment:
Since we are only interested in the total, would it make sense to limit the number of results? Passing `?s=1` or even `s=0`should be enough, so the backend does not have to serialize 10 records (as per default).
This request will probably not be performed very often but just in case.